packages/wrangler/src/api/startDevWorker/BundlerController.ts

```diff
@@ -80,6 +80,7 @@ export class BundlerController extends Controller<BundlerControllerEventMap> {
 			const entry: Entry = {
 				file: config.entrypoint,
 				projectRoot: config.projectRoot,
+				configPath: config.config,
```

Comment:
Why do we need `entry.configPath` if `config.config` is available?

Reply:
The config object is not available in findAdditionalModules. It felt cleaner to just add it to Entry rather than threading it around all over the place.